File: medplum-java/medplum-model/src/main/java/com/fhir/NutritionOrder_Nutrient.java
package com.fhir;

import com.StagedBuilderStyle;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import java.lang.String;
import java.util.List;
import java.util.Optional;
import org.immutables.value.Value;

@Value.Immutable
@StagedBuilderStyle
@JsonDeserialize(
    as = ImmutableNutritionOrder_Nutrient.class
)
@JsonIgnoreProperties(
    ignoreUnknown = true
)
public interface NutritionOrder_Nutrient {
  Optional<CodeableConcept> modifier();

  Optional<List<Extension>> modifierExtension();

  Optional<List<Extension>> extension();

  Optional<String> id();

  Optional<Quantity> amount();

  static ImmutableNutritionOrder_Nutrient.Builder builder() {
    return ImmutableNutritionOrder_Nutrient.builder();
  }
}

File: medplum-typescript/packages/sdk/src/model/fhir/NutritionOrder_Nutrient.ts
import { CodeableConcept } from "./CodeableConcept";
import { Extension } from "./Extension";
import { Quantity } from "./Quantity";

export interface NutritionOrder_Nutrient {
    id: string | null | undefined;
    extension: Extension[] | null | undefined;
    modifierExtension: Extension[] | null | undefined;
    modifier: CodeableConcept | null | undefined;
    amount: Quantity | null | undefined;
}
